ERCS/erc-x.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 7550
+title: Transient ERC-20 Approvals
+description: Transient ERC-20 approvals via function calls and EIP-712 secp256k1 signatures
+author: Joey Santoro (@joeysantoro), Moody Salem (@moodysalem)
+discussions-to: TODO
```

Comment:
Since you don't have a discussions thread, I'll leave this comment here:

> Would it be possible to support [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) signatures as well? Probably not the most necessary addition, but if it isn't too hard, it could be worth it.